slowmovie.py

```diff
@@ -212,6 +211,7 @@ def find_subtitles(file):
 parser.add_argument("-s", "--start", type=int, help="start playing at a specific frame")
 parser.add_argument("-c", "--contrast", default=1.0, type=float, help="adjust image contrast (default: %(default)s)")
 parser.add_argument("-l", "--loop", action="store_true", help="loop a single video; otherwise play through the files in the videos directory")
+parser.add_argument("-e", "--epd", default="waveshare_epd.epd7in5_V2", help="the name of the display device driver to use (default: %(default)s)")
```

Comment:
```suggestion
parser.add_argument("-e", "--epd", help="the name of the display device driver to use (default: %(default)s)")
```
This is keeping EPD type in `omni-epd.ini` from working. It might make more sense to have the default in the config file, either `slowmovie.conf` or `omni-epd.ini` (maybe those should have the same extension).

But then, if config files are missing, it'll print the list of displays instead.

Reply:
It does work the way it is. Might not even be worth messing with.

Reply:
So if we leave the `-e` option, but set the default to nothing, we'd have the following order of precedence. 

1. `slowmovie.py` checks for `-e`, if there that's the priority
2. `slowmovie.conf` - essentially the same as checking for `-e` just in the config file
3. `omni-epd.ini` - if nothing else attempting loading from INI file

So if the user deletes `omni-epd.ini`, or otherwise messes with it to screw up the EPD selection __and__ sets no default via the `slowmovie.conf` file or `-e` at the command line they should just get a "no driver found" error. Essentially they'll get a list of options. We could modify the error message to explain what they need to do? We could also re-create some files but if we set things to default and the display is not a 7.5in waveshare there is essentially the same problem of it not working right? I think if we set some defaults in the configs and they get messed with that's not really our fault. We aren't hurting existing installs that may update, to me that's the main thing.

Reply:
Sounds good.